src/main/cli/settings.c

```diff
@@ -1180,6 +1180,8 @@ const clivalue_t valueTable[] = {
     { "thrust_linear",              VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 150 }, PG_PID_PROFILE, offsetof(pidProfile_t, thrustLinearization) },
 #endif
 
+    { "airmode_transition_throttle",VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 100 }, PG_PID_PROFILE, offsetof(pidProfile_t, airmode_transitioned_throttle) },
```

Comment:
I would start with throttle in `airmode_throttle_transistion[ed]` as used in other commands but it's not fully consistent currently either.

Reply:
It could probably be named better, just need to think of the better name.

Reply:
Waiting for other reviewers opinion here. We also need to have solid evidence (video, logs, diagrams) as this code has been here for a lot of years as we should be very sure about making changes to flight related parameters.

Reply:
The name is very tricky.  
Airmode is set to OFF.  
Maybe `airmode_off_transition`.

Reply:
Currently it adds the dynamic airmode logic all the time when airmode is set to off, which is behaviour we may not want. However, I would like to see if its preferred over the currently airmode off logic. 

I don't quite like the name `airmode_off_transition` as it makes it sound like the point where airmode turns off. IDK what a "good" name for this would likely end up being, but I am totally open to suggestions.